rules/windows/execution_suspicious_short_program_name.toml

```diff
@@ -30,3 +30,14 @@ process where event.type in ("start", "process_started") and length(process.name
  length(process.name) == 5 and host.os.name == "Windows" and length(process.pe.original_file_name) > 5
 '''
 
+[[rule.threat]]
```

Comment:
maybe https://attack.mitre.org/techniques/T1036/003/ is more appropriate ?

Reply:
adjusted.